script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_7.txt

```diff
@@ -65,59 +65,59 @@ It seems that Akiha doesn't think well of me after I left the Tohno family.
 <Page20>
-"...Well, that is fine.
-If you are feeling well, then let us continue our conversation.
+"...Well, that's fine then.//「……まあいいでしょう。
+If you're feeling well, then let us finish our conversation.//　体調が良いのなら話を済ませましょう。
 Have you been informed of the details regarding the circumstances of your move?"
 <Page21>
-"No, I don't know anything about it. I was just told that 'You are to come back to the mansion.' I only read about the death of the old man in the newspaper."
+"No, I don't know the detailed situation. I was just told that 'You are to come back to the mansion.' I only read about the death of the old man in the newspaper."//「いや、詳しい事情は何も。いいから屋敷に帰ってこいって話だけだ。<槙久|おやじ>が亡くなった事は新聞で知ったけど」 in game, which does Shiki /actually/ say: old man or Makihisa?
 <Page22>
-When the head of a business group passes away, it is covered in the financial newspapers.//企業グループのトップに立っていた人物が亡くなれば、それぐらい経済新聞で取りあげられる。
-Even if he isn't informed by his relatives, a disowned son can learn of his parents' death from such sources.
-The news of Tohno Makihisa's death was delivered to the Arima family after his funeral as if it was none of my business.//Would that work better to convey how much of an outsider Shiki feels about his "father's" death?
+When the head of a business group passes away, it is covered in the financial columns.//企業グループのトップに立っていた人物が亡くなれば、それぐらい経済新聞で取りあげられる。
+Even if he isn't informed by his relatives, a disowned son can still learn of his parents' death from news sites.//　親戚から報せはなくても、勘当された息子はニュースサイトで親の不幸を知れる世の中だ。
+The news of Tohno Makihisa's death was delivered to the Arima family after his funeral. It was as if it was none of my business.
 <Page23>
 "...You have my apologies. It was my fault for not informing you about father."
 <Page24>
 Akiha quietly bows her head.
 Her smooth black hair is beautiful, ...but now is not the time to be admiring it. I shouldn't have said that earlier.
 <Page25>
 "You don't have to worry about that.
-It's not like the dead would have come back to life even if I had come. On the contrary, I'm glad I didn't meet any of my relatives."
+It's not like the dead would have come back to life if I was there. On the contrary, I'm glad I didn't have to see any of my relatives."//　俺が行ったって死人が生き返るわけでもないんだ。逆に、親戚筋に会わなくて助かった」
 Being disowned puts you in a complicated position in many ways.
 <Page26>
 "...I see.
 It makes me feel a little better to hear you say that."
 <Page27>
 The same goes for me, having a sister as frank as her.
-A funeral is a ritual for those who can't forget the deceased to lay their feelings to rest.
-For a son and a father who had severed their ties,
+Funerals are a ritual for those who cannot forget the deceased to lay their feelings to rest.
+For a father and son who severed their ties long before.//　とうの昔に関わりを断ってきた息子と父親。
```

Comment:
Not sure you want the "For a" on the front, this used to be conjoined with the one below but it's sorta just a statement. It also might read as a connection to the line above, when it's really the opposite implication (there are no feelings to lay to rest)

Reply:
maybe
> We are father and son with ties long since severed.